lib/dune

```diff
@@ -5,6 +5,7 @@
 (library
  (name lib)
  (libraries prelude mrmime threads netstring unix re)
+ (modules lib config)
  (inline_tests (backend qtest.lib)))
```

Comment:
We should get rid of this `inline_tests` expression, I think, until we put some `qtest`-y stuff into the project.  Unless I remove it, I get the following error:

```
> dune test
File "lib/.lib.inline-tests/inline_test_runner_lib.ml-gen", line 16, characters 5-11:
16 | open Config;;
          ^^^^^^
Error: Unbound module Config
............................       
Ran: 28 tests in: 0.10 seconds.
OK
```

Reply:
Done.

Reply:
As a note, I was only getting a warning, not an error. Not sure what the deal is there.

Reply:
Huh, thanks; useful information.  `dune` is sufficiently enigmatic that it's good to gather together our build error war stories.

Reply:
On the latest commit, `dune test` now works on my machine.